.github/workflows/ci.yml

```
@@ -208,6 +208,17 @@ jobs:
 
     steps:
       - uses: actions/checkout@v4
+
+      - name: Checkout Bevy repo for asset access
+        uses: actions/checkout@v4
+        with:
+          repository: 'bevyengine/bevy'
+          ref: 'latest'
+          path: 'bevy'
+
+      - name: Move Bevy assets to current workspace
+        run: mv bevy/assets ./
```

still failed, maybe this would work:
```suggestion
        run: mv bevy/assets ./static/
```